src/components/ConnectedMap/ConnectedMap.tsx

```diff
@@ -121,7 +120,7 @@ export function ConnectedMap({ className }: { className?: string }) {
 
   return (
     <Map
-      accessToken={accessToken}
+      accessToken={''}
```

Comment:
Do we need this prop anymore?

Reply:
Not needed, but this out of scope of this task